pyvista/plotting/lookup_table.py

```diff
@@ -76,14 +76,14 @@ def __setitem__(self, key, value):
             # internal VTK array
             self.table.Get().values = self
 
-    def __array_wrap__(self, out_arr, context=None):  # type: ignore[override]
+    def __array_wrap__(self, out_arr, context=None, return_scalar: bool = False):  # type: ignore[override]
```

Comment:
Needed due to numpy 2.0 deprecation here https://github.com/numpy/numpy/pull/25409